### DISCUSSION TITLE
Welcome to the Sunny Aggregator Forum!

### DISCUSSION THREAD

**Discussion by SuryaKhosla** [SunnyAggregator/governance#1] [Announcements]
Welcome to the Sunny Aggregator Forum! This is a place for creating and discussing proposals, feature requests, and anything else related to [Sunny Aggregator](https://sunny.ag).

If our admins notice that your noise level is high, you will be banned from our forum. So stay true to the purpose of this forum and focus on quality discussions.

Last but not least, be respectful. We will not tolerate any insults, racism, harassment or threats against members, even as a joke.

**Comment by brobyflanjoby** [comment]
I wrote in the Saber Discord that there should be a bribing function for veSBR tokens and was told that SUNNY was working on something like this. My thought was protocols could pay bribes to individuals for their "ve" tokens to help bring more incentives to their LPs which would also help bring those LPs liquidity. This is good for veSBR holders because it's a way for them to turn their governance voting power into a yield generating asset. This is a more impactful and less dilutive way for protocols to attract liquidity to their LPs.

**Comment by tmnxeq** [comment]
gm

a bit surprised to see the forum on github, but anyway good that we have it now

I assume the community is interested in the product roadmap & rough timelines (sunSBR, bribing etc) - anything to share?

**Reply by jimthereaper** [reply]
@SuryaKhosla Any alternatives other than GitHub as a discussion forum?

**Comment by sherriff11br** [comment]
I don´t know what to do, I Have one year of staking sol in my phantom wallet, conected it to the app in [https://sunsaberlp.com/claim](url) did the process, accepted the transaction in phantom, the page started loading, it got stuck and my founds disapeared, Where do I go now to get help???  traansaction details [https://solscan.io/tx/5tBZQaNnLLAi5yjh3PYMZKAGLoaeZDrnHdxy3SkGa2bxukSXFUke3zag7mtocJwbrJqthbCUe16HbYVXS1Sdgpc4](url)
